package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "prism-themes",
   "version": "1.3.0",
-  "description": "Addtional themes for the Prism syntax highlighting library.",
+  "description": "Additional themes for the Prism syntax highlighting library.",
   "main": "README.md",
   "scripts": {
-    "test": "echo \"Error: no tests required\" && exit 0"
+    "lint": "npx stylelint \"themes/*.css\"",
+    "lint-fix": "npx stylelint \"themes/*.css\" --fix",
```

Comment:
`npx` isn't required when running them from pkg.json scripts.

Reply:
I didn't know that. Thanks for pointing it out!